submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+But what about the risks?
+
+ 
+
+## Nuclear Safety in Factorio: A Brief History
+
+Nucelar power was something that Wube wanted to get around to for quite a while and they shared some early design ideas before releasing the version know in 0.15. When they shared their first implementation tinkers in [FFF#164](https://www.factorio.com/blog/post/fff-164), there was a lot of focus on heating and cooling water and steam. The early designs involved high-temperature steam being cooled and only partially consumed in turbines, then condensed back into water at 95C in cooling towers, and then being fed back into heat exchangers. You can imagine this being difficult to balance, leading perhaps to one thing or another getting stuck pretty quickly. After a few more weeks of working on it, Wube decided around [FFF#167](https://www.factorio.com/blog/post/fff-167) that dealing with so much temperature management can be avoided and that water and steam could simply be treated as two separate fluids with fixed temperatures, hence simplifying regular boilers too. 
+
+Meanwhile, the effects of temperature on the reactor itself were also softened: At first it was hinted that a nuclear reactor that heated up uncontrollably would explode, hence reactors would demand constant monitoring as in real life. This idea was softened before release though, such that a nuclear plants uses its internal emergency cooling systems to never let it itself heat up beyond 1000C, effectively allowing you to safely leave it alone. Well, mostly. Beware that it still *can* explode if is at over 900C and gets destroyed by a reckless player or biter. And if other reactors are close by, this may lead to [quite a light show](https://www.youtube.com/watch?v=l4ePIuhbiPg&ab_channel=Trupen). Despite the trivialized safety, we can see that Wube has provided a nuclear system which could still get pretty interesting and risky via the added complexity from the neighbor bonus. For those who would have preferred the high-risk and more realistic approach, there are multiple [mods]() that offer a look into the problems of meltdowns and radiation hazards.
+
+ Now that we' seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.

```

Comment:
We need some more media, and I think this is a perfect location for a small video of a nuclear explosion. Ideally you could record your own, or we could find a twitch clip or something. 

We could also embed Trupens video if you want.

Reply:
I link it already, so embedding it would be cleaner. I'll insert a request for it if I cannot manage myself